project/pom.xml

```diff
@@ -85,7 +85,7 @@
 			<plugin>
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>exec-maven-plugin</artifactId>
-				<version>3.1.0</version>
+				<version>3.0.0</version>
```

Comment:
I changed the Version from 3.1.0 to 3.0.0 because eclipse throws a `Plugin could not be resolved.` warning when using 3.1.0